mautic/mautic
# [UX] Change how the ways to implement a form are called

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
When you create a form, you are presented with 2 options to make it live: Automatic and Manual Copy. The problem is that both are manual and both require copying. What differs from them is the way the form will be hosted. 

In the first type, a link is generated that calls the Mautic external server to generate the form and returns it to the user's browser when he or she is accessing the page. In the second case, you do not depend on an external platform server to generate this form dynamically because the complete HTML will be placed directly in the content management system. 

This PR proposes to rename these options to reflect their true nature, reducing confusion for users who are setting up forms. This change will improve user experience by providing clearer instructions, ultimately making the form setup process more efficient.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms
3. Create a form
4. Save the form
5. Look at the alert card

![image](https://github.com/mautic/mautic/assets/116097999/f639e719-6a6a-428e-a819-d8868eea87be)
